qubes/mgmt.py

```diff
@@ -534,3 +570,95 @@ def vm_kill(self):
         assert not self.arg
         self.fire_event_for_permission()
         yield from self.dest.kill()
+
+    @api('mgmt.Events', no_payload=True)
```

Comment:
I take that back - broadcasting such events would be better - will not complicate handling methods when the tool doesn't care about progress, and will allow to display the progress by a separate tool (for example tray icon in GUI VM, even if backup operation triggered by Mgmt VM/cron).